jetty-servlet/src/test/java/org/eclipse/jetty/servlet/StatisticsServletTest.java

```diff
@@ -0,0 +1,174 @@
+//      http://www.eclipse.org/legal/epl-v10.html
+//
+//      The Apache License v2.0 is available at
+//      http://www.opensource.org/licenses/apache2.0.php
+//
+//  You may elect to redistribute this code under either of these licenses.
+//  ========================================================================
+//
+
+package org.eclipse.jetty.servlet;
+
+import org.eclipse.jetty.http.HttpTester;
+import org.eclipse.jetty.http.HttpVersion;
+import org.eclipse.jetty.server.LocalConnector;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.server.handler.StatisticsHandler;
+import org.eclipse.jetty.server.session.SessionHandler;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.xml.sax.InputSource;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.xml.xpath.XPath;
+import javax.xml.xpath.XPathFactory;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringReader;
+import java.nio.ByteBuffer;
+
+/**
+ *
+ */
+public class StatisticsServletTest
+{
+    private Server _server;
+
+    private LocalConnector _connector;
+
+    @Before
+    public void createServer()
+    {
+        _server = new Server();
+
+        _connector = new LocalConnector( _server );
+        _server.addConnector( _connector );
+
+    }
+
+
+    @After
+    public void destroyServer()
+        throws Exception
+    {
+        _server.stop();
+        _server.join();
+    }
+
+    @Test
+    public void get_stats()
+        throws Exception
+    {
+
+        StatisticsHandler statsHandler = new StatisticsHandler();
+        _server.setHandler(statsHandler);
+        ServletContextHandler statsContext = new ServletContextHandler(statsHandler, "/");
+
+        statsContext.addServlet( new ServletHolder( new TestServlet() ), "/test1" );
+
+        ServletHolder servletHolder = new ServletHolder( new StatisticsServlet() );
+        servletHolder.setInitParameter( "restrictToLocalhost", "false" );
+
+        statsContext.addServlet( servletHolder, "/stats" );
+
+        statsContext.setSessionHandler( new SessionHandler() );
+
+        _server.start();
+
+        getResponse("/test1" );
+
+        String response = getResponse("/stats?xml=true" );
+
+        Stats stats = parseStats( response );
+
+        Assert.assertEquals(1, stats.responses2xx);
+
+        getResponse("/stats?statsReset=true" );
+
+        response = getResponse("/stats?xml=true" );
+
+        stats = parseStats( response );
+
+        Assert.assertEquals(1, stats.responses2xx);
+
+        getResponse("/test1" );
+
+        getResponse("/nothing" );
+
+        response = getResponse("/stats?xml=true" );
+
+        stats = parseStats( response );
+
+        Assert.assertEquals(3, stats.responses2xx);
+
+        Assert.assertEquals(1, stats.responses4xx);
+
+    }
```

Comment:
Compare the method above with the one below. The method above is unreadable due to the blank lines between every statement. 
The method below groups the request lines, then a blank line, then the response lines - much more readable.
Please update the method above following a similar grouping logic: non-assert lines, then blank line, then assert lines.